scripts/pi-hole/php/groups.php

```diff
@@ -920,7 +920,11 @@ function JSON_error($message = null)
                 continue;
             }
 
-            if(preg_match("/[^a-zA-Z0-9:\/?&%=~._()-;]/", $address) !== 0) {
+            // this will remove first @ that is after schema and before domain
+          ￼ // $1 is optional schema, $2 is userinfo
+            $check_address = preg_replace("|([^:/]*://)?([^/]+)@|", "$1$2", $address, 1);
```

Comment:
Looks like False positive, changing comment to multi-line form is fixing the error.